test-helpers/src/connection/kafka/java.rs

```diff
@@ -35,7 +35,7 @@ impl KafkaConnectionBuilderJava {
         // The list of dependencies can be found here: https://repo1.maven.org/maven2/org/apache/kafka/kafka-clients/3.7.0/kafka-clients-3.7.0.pom
         // These are deployed to and loaded from a path like target/debug/jassets
         let jvm = Jvm::new(&[
-            "org.apache.kafka:kafka-clients:3.7.0",
+            "org.apache.kafka:kafka-clients:3.8.1",
```

Comment:
Will the above change be backward compatible with 3.7.0? Or are you saying that 3.7.0 will not use `RequestBody::DescribeCluster(_) at all?`

Reply:
The shotover changes in this PR are compatible with both kafka client 3.7 and 3.8
DescribeCluster is not sent in 3.7, it is only sent in 3.8 onwards.